api/src/functionalTest/java/uk/gov/hmcts/payment/functional/tokens/OneTimePasswordFactory.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.payment.functional.tokens;
+import org.jboss.aerogear.security.otp.Totp;
+import org.springframework.boot.test.context.TestComponent;
+
+@TestComponent
+public class OneTimePasswordFactory {
```

Comment:
Can these be utility class? so that this can be extracted as an independent component later? this can be used as a good functionality later one. Sateesh is working on gnerating OTP using an endpoint. can this be moved there by any chance?

Reply:
in nutshell we want to move it out of tests code to a util code

Reply:
Ok I will make this as an actual component.